python/main.py

```diff
@@ -28,23 +28,24 @@ def route_webhook():
     data = json.loads(request.data.decode('utf8'))
     settings.log.debug('webhook type received %s', data['type'])
     if data.get('type') == 'transaction.created':
-        return jsonify(create_transaction(data['data'], settings, 0))
+        body, code = create_transaction(data['data'], settings, 0)
+        return jsonify(body), code
     else:
         settings.log.warning('Unsupported webhook type: %s', data['type'])
-        return jsonify({'error': 'Unsupported webhook type'})
+        return jsonify({'error': 'Unsupported webhook type'}), 400
 
 
 def create_transaction(data, settings, expected_delta):
     # Sync the account so we get the latest payees
     ynab_client.sync()
 
     if data['amount'] == 0:
-        return jsonify({'error': 'Amount is 0'})
+        return {'error': 'Monzo transaction account is 0.'}, 400
```

Comment:
I am not sure about this. Returning an error here will cause Monzo to keep retrying. Their webhook is valid, but we just dont want to forward it.